packages/calling/src/CallingClient/line/line.test.ts

```diff
@@ -154,4 +157,111 @@ describe('Line Tests', () => {
       expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
     });
   });
+  describe('Line calling tests', () => {
+    let line;
+
+    beforeEach(() => {
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.INFO
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.clearAllMocks();
+      jest.useRealTimers();
+      line.removeAllListeners();
+    });
+    it('Return a successful call object while making call', () => {
+      const createCallSpy = jest.spyOn(line.callManager, 'createCall');
+      const call = line.makeCall({address: '5003', type: CallType.URI});
+
+      expect(createCallSpy).toBeCalledOnceWith(
+        {address: 'tel:5003', type: 'uri'},
+        CallDirection.OUTBOUND,
+        undefined,
+        line.lineId
+      );
+      expect(call).toBeTruthy();
+      expect(line.getCall(call ? call.getCorrelationId() : '')).toBe(call);
+      expect(call ? call.direction : undefined).toStrictEqual(CallDirection.OUTBOUND);
+      call?.end();
+    });
+
+    it('Return a successful call object while making call to FAC codes', () => {
```

Comment:
In this case and the below cases also where the call object is truthy, won't createCall will be invoked? If yes, please add the spy invocation check in those as well

Reply:
Yes createCall will be invoked. Added checks for createCall